src/test/kotlin/reactor/kotlin/adapter/rxjava/RxJava2AdapterExtTest.kt

```diff
@@ -97,16 +97,17 @@ class RxJava2AdapterExtTest {
                 .assertNotComplete()
     }
 
-    @Test
-    fun `Single to Mono`() {
-        Single.just(1)
-                .toMono()
-                .test()
-                .expectFusion(Fuseable.ANY, Fuseable.ASYNC)
-                .expectNext(1)
-                .expectComplete()
-                .verify()
-    }
+// FIXME
+//    @Test
+//    fun `Single to Mono`() {
+//        Single.just(1)
+//                .toMono()
+//                .test()
+//                .expectFusion(Fuseable.ANY, Fuseable.ASYNC)
+//                .expectNext(1)
+//                .expectComplete()
+//                .verify()
+//    }
```

Comment:
the test can be kept around to validate onNext/onComplete are translated, but simply removing the assertion of fusion negotiation:
```suggestion
    @Test
    fun `Single to Mono`() {
        Single.just(1)
                .toMono()
                .test()
                .expectNext(1)
                .expectComplete()
                .verify()
    }
```
Note also that the `Fuseable` trait removal will be dealt with via the reactor-addons issue.

Reply:
fixed